### PR TITLE
article additions

### PR DESCRIPTION
**Pull request by sibrooke**

### CODE REVIEW

File: wix-events-v2/wix-events-v2/Orders/guides/orders sort.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Orders: Supported Sorting and Faceting
+
+Select functions allow sorting results by field. Use `field:asc` to sort results in **ascending** order, and `field:desc` to sort in **descending** order.
+
+For example, to sort ticket definitions by created date in ascending order, and then by name in descending order, use this format in the `options` parameter: 
+
+```json
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/Orders/guides/orders sort.md:7]: in this case, it's not json, it's js

```suggestion
```js
```

File: wix-events-v2/wix-events-v2/Orders/guides/orders sort.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Orders: Supported Sorting and Faceting
+
+Select functions allow sorting results by field. Use `field:asc` to sort results in **ascending** order, and `field:desc` to sort in **descending** order.
+
+For example, to sort ticket definitions by created date in ascending order, and then by name in descending order, use this format in the `options` parameter:
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/Orders/guides/orders sort.md:5]: code font for field names

File: wix-events-v2/wix-events-v2/Orders/guides/orders sort.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+For example, to sort ticket definitions by created date in ascending order, and then by name in descending order, use this format in the `options` parameter: 
+
+```json
+{
+  sort: 'created:asc,name:desc'
+}  
+```
+
+The following table shows property support for filters and sorting
+for the Orders object:
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/Orders/guides/orders sort.md:14]: Orders object or Order object?

File: wix-events-v2/wix-events-v2/Orders/guides/orders sort.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+| `created`          | Sortable |           |
+| `name`             | Sortable |           |
+| `score`            | Sortable |           |
+| `orderNumber`      | Sortable |           |
+| `status`           |          | Facetable |
+| `archived`         |          | Facetable |
+| `fullyCheckedIn`   |          | Facetable |
+
+__Related content:__
+[`listOrders`](https://www.wix.com/velo/reference/wix-events-v2/orders/listorders)
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/Orders/guides/orders sort.md:27]: relative link, not absolute here

File: wix-events-v2/wix-events-v2/guides/partial-updates.md [hunk 5]
```diff
@@ -1,14 +1,16 @@
 # Partial Updates
-Most update endpoints support partial updates via a combination of a _partial entity body_ and a `fields` parameter.
+Most update functions support partial updates via a combination of a _partial entity body_ and a `fields` parameter.
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/guides/partial-updates.md:2]: We generally don't italicize unless it's really necessary for emphasis. I think you can use standard formatting here.

File: wix-events-v2/wix-events-v2/guides/partial-updates.md [hunk 6]
```diff
@@ -1,14 +1,16 @@
...
 
 ## Fields parameter
-The `fields` parameter accepts a set of field paths that specifies the entity data to update. In general, the fields path begins from the root of the request body.
+The `fields` parameter accepts a set of field paths that specifies the entity data to update. 
 
-When the `fields` parameter contains a sub-path of the entity fields, like `event` - all nested fields are included in the update: `event.title`, `event.description`, etc.
+The field path begins from the root of the request body and may contain several levels of fields separated by dot notation. This indicates the specific field for update.
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/guides/partial-updates.md:7]: > root of the request body

the concept of "request body" is a REST term, not Velo.

> separated by dot notation

The fields paths _use_ dot notation or are _in_ dot notation.

> the specific field for update

specific ~~field for~~ **fields to** update

File: wix-events-v2/wix-events-v2/guides/partial-updates.md [hunk 7]
```diff
@@ -1,14 +1,16 @@
...
 
 ## Fields parameter
-The `fields` parameter accepts a set of field paths that specifies the entity data to update. In general, the fields path begins from the root of the request body.
+The `fields` parameter accepts a set of field paths that specifies the entity data to update. 
 
-When the `fields` parameter contains a sub-path of the entity fields, like `event` - all nested fields are included in the update: `event.title`, `event.description`, etc.
+The field path begins from the root of the request body and may contain several levels of fields separated by dot notation. This indicates the specific field for update. 
 
-When the `fields` parameter is empty, the request will modify ALL entity fields.
+For example, if the field path is `f.a.b`, `f` represents a field in the request root, `a` is a sub-field in the `f` field and `b` is a sub-field in `f.a`.
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/guides/partial-updates.md:9]: We avoid using generic terms like `f.a.b.`. Instead, use real examples from your API.

File: wix-events-v2/wix-events-v2/guides/partial-updates.md [hunk 8]
```diff
@@ -1,14 +1,16 @@
...
-The `fields` parameter accepts a set of field paths that specifies the entity data to update. In general, the fields path begins from the root of the request body.
+The `fields` parameter accepts a set of field paths that specifies the entity data to update. 
 
-When the `fields` parameter contains a sub-path of the entity fields, like `event` - all nested fields are included in the update: `event.title`, `event.description`, etc.
+The field path begins from the root of the request body and may contain several levels of fields separated by dot notation. This indicates the specific field for update. 
 
-When the `fields` parameter is empty, the request will modify ALL entity fields.
+For example, if the field path is `f.a.b`, `f` represents a field in the request root, `a` is a sub-field in the `f` field and `b` is a sub-field in `f.a`.       
+
+When the `fields` parameter contains a sub-path of the entity fields, like `event`, all nested fields are included in the update, for example, `event.title`, `event.description`.
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/guides/partial-updates.md:11]: I'm having trouble understanding what this means

File: wix-events-v2/wix-events-v2/guides/partial-updates.md [hunk 9]
```diff
@@ -1,14 +1,16 @@
...
 
-`fields` parameter behavior follows [google.protobuf.FieldMask][google-protobuf-fieldmask] semantics.
+When the `fields` parameter is empty, the request will modify ALL entity fields.
```
Comment (imnotashrimp) [inline wix-events-v2/wix-events-v2/guides/partial-updates.md:13]: > will modify ALL

- use present tense
- don't use all-caps, standard formatting is fine here

File: wix-events-v2/wix-events-v2/guides/partial-updates.md (hunk 7 above)
Comment (sibrooke) [reply wix-events-v2/wix-events-v2/guides/partial-updates.md]: Fixed but not sure if it's too clunky now

File: wix-events-v2/wix-events-v2/guides/partial-updates.md (hunk 8 above)
Comment (sibrooke) [reply wix-events-v2/wix-events-v2/guides/partial-updates.md]: Tried to clarify. See if it's better now.

**Pull request closed without merging**